packages/preview/uo-dissertation-2024/1.0.0/template/main.typ

```diff
@@ -0,0 +1,147 @@
+// main.typ - Main dissertation file
+// University of Oregon Dissertation Template
+
+// Import configuration and styling
+// #import "../config.typ": * // for local config file
+// #import "@local/uo-dissertation:1.0.0/config.typ" // for local package
+#import "@preview/uo-dissertation-2024:1.0.0/config.typ": * // for preview package on typst universe
```

Comment:
You should remove the `/config.typ` part here. When importing a package, you always import the file defined as the `entrypoint` in the manifest, and it is not possible to import any other file. Here this is what you want, since your entry point is `config.typ`. But in general, if you want to expose functions and variables from other files you would have to re-export them in your entry file.